README.md

```diff
@@ -0,0 +1,115 @@
+# Adam Impact Study
+
+## Description
+This package includes all the functions and utilities needed to run the impact study.
+
+## Installation
+
+### Using PDM (recommended)
+To install the necessary dependencies using PDM, run:
+
+```bash
+pdm install
+```
+
+This command will install all dependencies as specified in the `pyproject.toml` file.
+
+### Using pip
+If you do not have PDM installed, you can alternatively install dependencies using pip:
+
+```bash
+pip install -r requirements.txt
+```
+
+However, using PDM is recommended to ensure consistent and reproducible environments.
+
+### Download Ephemeris Files
+
+To download the assist ephemeris files, run:
+
+```
+python set_up_assist.py
```

Comment:
To run this command with the virtualenv python, it looks like you need to activate it. If users are only using pdm, you will need to do something like `eval $(pdm venv activate)`